wazuh/wazuh-kibana-app
# Refactored events documents discover links building

Hi team,

this PR refactors events documents links to avoid using a resolver with internal re-directions. This allows to disable the overview plugin, as it doesn't rely on that plugin to make re-directions.

Closes #3850 

To test it:

- Go to `Security Events -> Events`
- Click on a document row to open it
- Check `View surrounding documents` & `View single document` links are pointing to Discover plugin _(not Wazuh)_
- Check `View surrounding documents` & `View single document` are loading properly

![image](https://user-images.githubusercontent.com/9343732/156229557-752aed51-407f-4bdb-bb93-b06b9ce1a503.png)


- Click on a field under **_Selected fields_** section
- Check Visualize button is loading properly
![image](https://user-images.githubusercontent.com/9343732/156229402-f9d95e86-a307-4750-9472-7d31e7ce270c.png)
